content/concepts/v4-nft.md

```diff
@@ -0,0 +1,42 @@
+---
```

Comment:
please remove version references from all URLs. It makes it harder moving forward with a next version if we do that. We also dont have our old stuff under `v2` URLs